owncloudDomain/src/test/java/com/owncloud/android/domain/files/model/OCFileTest.kt

```diff
@@ -18,82 +18,178 @@
  */
 package com.owncloud.android.domain.files.model
 
-import com.owncloud.android.testutil.OC_ACCOUNT_NAME
-import org.junit.Assert
+import com.owncloud.android.testutil.OC_FILE
```

Comment:
Yes, we have both since files are the most common datamodel in the app. This way we won't need to create a copy of OC_FILE each time we want a folder.

Reply:
I mean calling it `DUMMY_OC_FILE` instead of `OC_FILE`...

Reply:
We call `OC_` instead of `DUMMY_`. We did that for capabilities and shares too.